src/form/demos/enUS/index.demo-entry.md

```diff
@@ -64,7 +65,8 @@ show-label
 | rule-path | `string` | `undefined` | The path to get rules from the parent form's rule object. If not set, it will use the path of the parent form item instead. |
 | show-feedback | `boolean` | `true` | Whether to show the feedback area. |
 | show-label | `boolean` | `true` | Whether to show a label. If not set, it will inherit `show-label` from the parent form. |
-| show-require-mark | `'left' \| 'right' \| 'boolean'` | `'right'` | Whether to show required symbol. If not set, it will use `show-require-mark` from the parent form. |
+| show-require-mark | `boolean` | `-` | Whether to show required symbol. If not set, it will use `show-require-mark` from the parent form. |
+| require-mark-placement | `'left' \| 'right'` | `'right'` | Require Mark placement. If not set, it will use `require-mark-placement` from the parent form. |
```

Comment:
```suggestion
| require-mark-placement | `'left' \| 'right'` | `'right'` | Require mark placement. If not set, it will use `require-mark-placement` from the parent form. |
```